src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/message/SplitTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact.message;
+
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.api.junit.jupiter.SoftAssertionsExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split.byColon;
+import static uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split.byPlus;
+import static uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split.bySegmentTerminator;
+
+@ExtendWith(SoftAssertionsExtension.class)
+class SplitTest {
+    @Test
+    public void When_SplittingBySegmentTerminator_Expect_CorrectResult(SoftAssertions softly) {
```

Comment:
The `public` access modifiers are not needed for Junit5

Reply:
Removed the access modifiers